src/applications/personalization/profile/components/personal-information/ConfirmRemoveModal.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Modal from '@department-of-veterans-affairs/component-library/Modal';
+import { toLower } from 'lodash';
```

Comment:
Can we just use `.toLowerCase()` directly on the string?

Reply:
yep yes we can